packages/dev-server/src/middleware/__tests__/JSInspectorMiddleware-test.ts

```diff
@@ -0,0 +1,167 @@
+import { ChildProcess } from 'child_process';
+import type { IncomingMessage, ServerResponse } from 'http';
+import fetch from 'node-fetch';
+import open from 'open';
+import { URL } from 'url';
+
+import JSInspectorMiddleware from '../JSInspectorMiddleware';
+
+jest.mock('fs');
+jest.mock('node-fetch');
+jest.mock('open');
+jest.mock('rimraf');
+jest.mock('temp-dir', () => '/tmp');
+
+const { Response } = jest.requireActual('node-fetch');
+
+describe('JSInspectorMiddleware', () => {
+  it('should return specific app entity for GET request with given applicationId', async () => {
+    const appId = 'io.expo.test.devclient';
+    const entities = JSON.parse(RESPONSE_FIXTURE) as { [key: string]: string }[];
+    const entity = entities.find(object => object.description === appId);
+
+    const req = createRequest(`http://localhost:8081/inspector?applicationId=${appId}`);
+    const res = createMockedResponse();
+    const next = jest.fn();
+    const mockFetch = fetch as jest.MockedFunction<typeof fetch>;
+    mockFetch.mockReturnValue(Promise.resolve(new Response(RESPONSE_FIXTURE)));
+
+    const middlewareAsync = JSInspectorMiddleware();
```

Comment:
ya this looks incorrect without the new keyword, so changing the method to lowercase would make the most sense imo.